### DISCUSSION TITLE
Typescript Errors

### DISCUSSION THREAD

**Discussion by litewarp** [hayes/pothos#76] [Q&A] [answered]
So only two remaining typescript errors:

1. Implementing interfaces on loadableObject or loadableNode. If I try and add an interface to loadableObject or loadableNode, I get the following error: 

![image](https://user-images.githubusercontent.com/1900476/120147007-d8b81880-c19a-11eb-8193-1082bdd849b1.png)

2. Shape of "connection":  I still get the error on the connection type. 

**SOLVED** - I needed to set `"strict": true` in my tsconfig.  Glad I read [this issue](https://github.com/hayes/giraphql/issues/44#issuecomment-830461414). Docs should definitely identify that.

**Comment by hayes** [comment, marked as answer]
Glad you figured out the strict mode thing.  I've updated docs to mention that explicitly, and will try to remember it as a debugging tip for the future.  Still need to try to understand specifically what TS feature causes that behavior. 

The interface issue was a missing feature, I've released a new version of the dataloader plugin (along with a non preview version of the relay changes) that should make interfaces work correctly with loadableObject and loadableNode
